config/config.go

```diff
@@ -246,3 +246,13 @@ func (c *Config) GetWarpQuorum(blockchainID ids.ID) (WarpQuorum, error) {
 	}
 	return WarpQuorum{}, errFailedToGetWarpQuorum
 }
+
+// Config implempents the peers.Config interface
+func (c *Config) GetPChainAPI() *APIConfig {
+	return c.PChainAPI
+}
+
+// Config implempents the peers.Config interface
```

Comment:
should we add a check that this type implements the peers.Config interface

Reply:
The compilation will fail if it doesn't since we pass this config into AppRequestsNetwork which requires peers.Config interface

Reply:
makes sense and is a nit, but can we still add it? We have it in other places of the repo, and believe it's a standard go practice

Reply:
Actually can't do it here since it would introduce a circular dependency. I'm happy to enter an issue for cleaning up some of the config handling between both this and `awm-relayer` but I'd like to punt on it until a future PR

Reply:
good with deferring, can we create an issue to track and ref here